lib/logstash/filters/metrics.rb

```diff
@@ -193,7 +195,7 @@ def flush(options = {})
     return unless should_flush?
 
     event = LogStash::Event.new
-    event.set("message", Socket.gethostname)
+    event.set("host", @host)
```

Comment:
I can do that if you wish. But storing the hostname in a field named message is more a bug than a feature to me.

Shall I split this PR to place the field renaming in another one?